src/api/coupon-config/content-types/coupon-config/schema.json

```diff
@@ -52,7 +52,7 @@
         "ES - Spain",
         "SE - Sweden",
         "CH - Switzerland",
-        "UK - United Kingdom",
+        "GB - United Kingdom",
```

Comment:
Do all the codes used here match the same ISO standard used in `payments-next`?

Reply:
Yep, only United Kingdom was different